src/merkle/tiered_smt/tests.rs

```diff
@@ -66,24 +66,32 @@ fn tsmt_insert_two_16() {
     let mut tree_root = get_init_root();
     let index_a = NodeIndex::make(32, raw_a >> 32);
     let leaf_node_a = build_leaf_node(key_a, val_a, 32);
-    tree_root = store.set_node(tree_root, index_a, leaf_node_a.into()).unwrap().root;
+    tree_root = store
+        .set_node(tree_root.into(), index_a, leaf_node_a.into())
+        .unwrap()
+        .root
+        .into();
```

Comment:
Would changing the return type of `get_init_root()` to `RpoDigest` and of `build_leaf_node()` to `Word` make this a bit simpler?

Reply:
I think it might be simpler to keep return type of `build_leaf_node()` as `RpoDigest` but change for `get_init_root()`.